net.newpipe.NewPipe.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>net.newpipe.NewPipe</id>
+  <name>NewPipe</name>
+  <summary>Free and lightweight YouTube frontend for Android</summary>
+  <project_license>GPL-3.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="vcs-browser">https://github.com/TeamNewPipe/NewPipe</url>
+  <url type="homepage">https://newpipe.net/</url>
+  <url type="bugtracker">https://gitlab.com/android_translation_layer/android_translation_layer/-/issues</url>
+  <url type="donation">https://newpipe.net/donate/</url>
+  <url type="translate">https://hosted.weblate.org/engage/newpipe/</url>
+  <description>
+    <p><em>NOTE:</em> This is an unofficial and experimental Flatpak build based on Android Translation Layer. Please report bugs to the ATL bug tracker instead of the NewPipe bug tracker https://gitlab.com/android_translation_layer/android_translation_layer/-/issues.</p>
+    <p>NewPipe does not use any Google framework libraries, or the YouTube API. It only parses the website in order to gain the information it needs. Therefore this app can be used on devices without Google Services installed. Also, you don't need a YouTube account to use NewPipe, and it's FLOSS.</p>
+    <p>ATL is a lightweight WINE like compatibility layer for running Android applications on GNU/Linux Freedesktop systems. With the help of ATL, NewPipe integrates nicely into the Linux Desktop environment. Including support for notification, media controls and automatic dark/light theme selection. The translation layer is already bundled into the Flatpak and and does not need to be installed separately.</p>
```

Comment:
I don't think the para about ATL is necessary, describe what Newpipe is - privacy-preserving YT frontend etc.

Reply:
I have removed the ATL para and instead added the list of supported services from their website